controller/users_blackbox_test.go

```diff
@@ -69,6 +71,38 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, controller
 }
 
+func (s *TestUsersSuite) TestEmailVerifiedOK() {
+	// given
+	user := s.createRandomUser("TestUpdateUserOK")
+	identity := s.createRandomIdentity(user, account.KeycloakIDP)
+	test.ShowUsersOK(s.T(), nil, nil, s.controller, identity.ID.String(), nil, nil)
+
+	// when
+	newEmail := "TestUpdateUserOK-" + uuid.NewV4().String() + "@email.com"
+	newFullName := "TestUpdateUserOK"
+	newImageURL := "http://new.image.io/imageurl"
+	newBio := "new bio"
+	newProfileURL := "http://new.profile.url/url"
+	newCompany := "updateCompany " + uuid.NewV4().String()
+	secureService, secureController := s.SecuredController(identity)
+
+	contextInformation := map[string]interface{}{
+		"last_visited": "yesterday",
+		"space":        "3d6dab8d-f204-42e8-ab29-cdb1c93130ad",
+		"rate":         100.00,
+		"count":        3,
+	}
+	//secureController, secureService := createSecureController(t, identity)
+	updateUsersPayload := createUpdateUsersPayload(&newEmail, &newFullName, &newBio, &newImageURL, &newProfileURL, &newCompany, nil, nil, contextInformation)
+	test.UpdateUsersOK(s.T(), secureService.Context, secureService, secureController, updateUsersPayload)
+
+	codes, err := s.Application.VerificationCodes().Query(account.VerificationCodeWithUser(), account.VerificationCodeFilterByUserID(user.ID))
+	require.Nil(s.T(), err)
+	require.Len(s.T(), codes, 1)
+	verificationCode := codes[0].Code
+	test.VerifyEmailUsersOK(s.T(), secureService.Context, secureService, secureController, verificationCode)
+}
+
```

Comment:
Add some very simple tests (no users, no codes) to check if it fails when we verify unknown code.

Reply:
50cd526433557dc0ac357bcb468bd2862b6ac833 thanks,done.